### PR TITLE
Completed shifts and Manage Volunteer shifts filter implemented

### LINKED ISSUE #829: Rearange Completed Shifts Page
## Description
As a USER,
I need TO keep ongoing and past shifts in completed shifts page,
so that I can log or edit the hours.

### Update [Required]
- [ ] Remove clear hours option
- [ ] Show Log Hours option

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6 hours

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
The shifts are filtered properly now on completed shifts page and manage volunteer shifts page.

Fixes #829 

# Type of Change:
- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
image
image

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: vms/pom/locators/upcomingShiftsPageLocators.py [hunk 1]
```diff
@@ -5,8 +5,7 @@ class UpcomingShiftsPageLocators(object):
     SHIFT_STIME_PATH = '//table//tbody//tr[1]//td[3]'
     SHIFT_ETIME_PATH = '//table//tbody//tr[1]//td[4]'
     SHIFT_CANCEL_PATH = '//table//tbody//tr[1]//td[5]'
-    LOG_SHIFT_HOURS_PATH = '//table//tbody//tr[1]//td[5]'
-
     INFO_BOX = 'alert-info'
-    CONTAINER = '//table'
     SUBMIT_PATH = '//form[1]'
+    CONTAINER = '//table'
```
Comment (Monal5031) [inline vms/pom/locators/upcomingShiftsPageLocators.py]: Why this change?

File: vms/pom/locators/manageShiftPageLocators.py [hunk 2]
```diff
@@ -1,8 +1,12 @@
 class ManageShiftPageLocators(object):
 
     CANCEL_SHIFT_PATH = '//table//tbody//tr[1]//td[5]'
+    CLEAR_SHIFT_HOURS_TEXT = 'html/body/div[2]/form/div/div[1]/h3'
```
Comment (Monal5031) [inline vms/pom/locators/manageShiftPageLocators.py]: we should `//` if we are using any index after a tag.

File: vms/pom/pages/completedShiftsPage.py [hunk 3]
```diff
@@ -20,46 +20,66 @@ def go_to_completed_shifts(self):
...
-        self.log_shift_timings(stime, etime)
-
     def log_shift_timings(self, stime, etime):
         self.element_by_xpath(self.elements.START_TIME_FORM).clear()
         self.send_value_to_xpath(self.elements.START_TIME_FORM, stime)
         self.element_by_xpath(self.elements.END_TIME_FORM).clear()
         self.send_value_to_xpath(self.elements.END_TIME_FORM, etime)
         self.submit_form()
 
+    def edit_hours(self, stime, etime):
```
Comment (Monal5031) [inline vms/pom/pages/completedShiftsPage.py]: Why this change?

File: vms/pom/pages/upcomingShiftsPage.py [hunk 4]
```diff
@@ -49,3 +46,5 @@ def get_cancel_shift(self):
     def cancel_shift(self):
         self.element_by_xpath(self.elements.SHIFT_CANCEL_PATH + "//a").click()
 
+    def get_result_container(self):
```
Comment (Monal5031) [inline vms/pom/pages/upcomingShiftsPage.py]: why this change?

File: vms/pom/locators/manageShiftPageLocators.py (hunk 2 above)
Comment (anjali-dhanuka) [reply vms/pom/locators/manageShiftPageLocators.py]: I just moved it, didn't write it. I'll update it.

Comment (Monal5031) [reply vms/pom/locators/manageShiftPageLocators.py]: okay.

**Comment by anjali-dhanuka** [comment]
Hi @kamsuri @nida @abhishekarya286 @Monal5031 Please review

File: vms/shift/tests/test_manageVolunteerShift.py [hunk 5]
```diff
@@ -545,6 +545,35 @@ def test_edit_request_email_volunteer(self):
         self.assertEqual(msg.subject, "Log Hours Edited")
         self.assertEqual(msg.to, ['volunteer@volunteer.com'])
 
+    def test_clear_hours(self):
+        """
+        Test clearing of shift hours.
+        """
+        sign_up_page = self.sign_up_page
```
[the remark this answers is not in the document]
Comment (Monal5031) [reply vms/shift/tests/test_manageVolunteerShift.py]: This `sign_up_page` is unused, remove it.

**Comment by anjali-dhanuka** [comment]
@abhishekarya286 Can you have a look at it again, the travis build was working fine the last time it ran. I guess we can merge this.
